src/Displaz.jl

```diff
@@ -234,28 +251,36 @@ When plotting lines, the `linebreak` keyword argument can be used to break the
 position array into multiple line segments.  Each index in the line break array
 is the initial index of a line segment.
 """
-function plot3d(plotobj::DisplazWindow, position; color=[1,1,1], markersize=[0.1], markershape=[0],
+function plot3d(plotobj::DisplazWindow, position; normal=[0,0,1], color=[1,1,1],
+                markersize=[0.1], markershape=[0], shader="generic_points.glsl",
```

Comment:
A problem with adding the shader in here is that displaz doesn't currently support per-geometry shaders, though it's probably a thing which *should* be supported.  The underlying difficulty is providing a UI to edit shader parameters in a nice way when there's a lot of different shaders loaded.